opentelemetry/src/sdk/export/trace/mod.rs

```diff
@@ -1,16 +1,14 @@
 //! Trace exporters
-use std::fmt::Debug;
-use std::sync::Arc;
-use std::time::SystemTime;
-
-use async_trait::async_trait;
-#[cfg(feature = "serialize")]
-use serde::{Deserialize, Serialize};
-
 use crate::{
     sdk,
     trace::{Event, Link, SpanContext, SpanId, SpanKind, StatusCode, TraceError},
 };
+use async_trait::async_trait;
+#[cfg(feature = "serialize")]
+use serde::{Deserialize, Serialize};
+use std::fmt::Debug;
+use std::sync::Arc;
+use std::time::SystemTime;
```

Comment:
I usually prefer to separate std, third-party, and crate-local imports, and I think that's a common way of doing things. Is grouping everything the common style here?

Reply:
oh this was actually not intentional, but it does seem like a standard grouping formatting in rustfmt would be nice.

Reply:
reverted